src/panels/config/storage/ha-config-section-storage.ts

```diff
@@ -148,7 +148,8 @@ class HaConfigSectionStorage extends LitElement {
                 >
                   ${isHAOS
                     ? html`${this.hass.localize(
-                          "ui.panel.config.storage.network_mounts.not_supported.haos"
+                          "ui.panel.config.storage.network_mounts.not_supported.os",
+                          { version: "10.0" }
```

Comment:
```suggestion
                          { version: "10.2" }
```
Ideally we would require 10.2, as that makes for a better experience, but that currently does not exist.

Reply:
yea. I also have to put the corresponding restriction in supervisor. Same issue there

Reply:
10.2 is now in beta.